tests/lib/nested.sh

```diff
@@ -87,7 +87,7 @@ nested_check_unit_stays_active() {
 nested_check_boot_errors() {
     local cursor=$1
     # Check if the service started and it is running without errors
-    if nested_is_core_20_system && ! nested_check_unit_stays_active "$NESTED_VM" 15 1; then
+    if (nested_is_core_20_system || nested_is_core_22_system) && ! nested_check_unit_stays_active "$NESTED_VM" 15 1; then
```

Comment:
Nested tests for uc22 are included in another PR #11456

Reply:
Do you want me to revert all changes in `tests/lib/nested.sh`?

Reply:
@valentindavid hi, yes please

Reply:
Once #11456 is merged, I'll create a new pr with other tests to be executed in uc22

Reply:
I have removed changes to `tests/lib/nested.sh`